index.html

```diff
@@ -65,20 +65,20 @@
     </linkedin>
     </social_icons>
 
-    <headline>
-        <primary_text>Hey There, I'm Alex.</primary_text>
-        <supporting_text>I'm a Software Developer.</supporting_text>
-        <description>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</description>
+    <headline id="headline">
+        <primary_text id="primary_text">Hey There, I'm Alex.</primary_text>
+        <supporting_text id="supporting_text">I'm a Software Developer.</supporting_text>
+        <description id="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</description>
```

Comment:
- [ ] Kindly I recommend you to use `p` tag paragraph (e.g., "I can help you build a product..."), This way will be semantically correct and the screen readers and other technologies will thank you

Reply:
used <p> tag for all plain text as requested, changed font sizing and needed some reformatting.